cdist/conf/type/__letsencrypt_cert/parameter/default/admin-email

```diff
@@ -0,0 +1 @@
+root@localhost
```

Comment:
We could put "honeypot@evil-domain.org" here - just kidding, but it would work....